Azure/sonic-buildimage
# Move RADV fastboot handling to a service script

**- Why I did it**
Fastboot script contains sections of RADV handling which are better suited in a separate service script.
Other services (swss, bgp, syncd) already have their service scripts. RADV is an addition to the list. 

**- How I did it**
Added start, stop sections for RADV service. Added handlers for special handling for fast, warm and cold reboot.

**- How to verify it**
Started the service and stopped the service successfully.
Performed warm and fast reboot without any issues (utility script unchanged).

`Stop Path:`
```
$ sudo /usr/local/bin/radv.sh stop

$ sudo systemctl status radv
â— radv.service - Router advertiser container
   Loaded: loaded (/lib/systemd/system/radv.service; enabled-runtime; vendor preset: enabled)
   Active: activating (auto-restart) since Fri 2020-08-07 19:58:36 UTC; 2s ago
  Process: 3018 ExecStart=/usr/bin/radv.sh wait (code=exited, status=0/SUCCESS)
  Process: 5545 ExecStop=/usr/bin/radv.sh stop (code=exited, status=0/SUCCESS)
 Main PID: 3018 (code=exited, status=0/SUCCESS)

$ sudo grep -i radv /var/log/syslog
Aug  7 19:42:05.378372 str-s6100-acs-4 NOTICE admin: Stopping radv service...
Aug  7 19:42:05.798310 str-s6100-acs-4 NOTICE admin: Warm boot flag: radv false.
Aug  7 19:42:05.805479 str-s6100-acs-4 NOTICE admin: Fast boot flag: radv false.
Aug  7 19:42:06.444666 str-s6100-acs-4 INFO radv.sh[7908]: 0
Aug  7 19:42:06.453780 str-s6100-acs-4 NOTICE admin: Stopped radv service...
```

`Start Path:`
```
$ sudo /usr/local/bin/radv.sh start

$ sudo grep -i radv /var/log/syslog
Aug  7 19:42:24.116816 str-s6100-acs-4 NOTICE admin: Starting radv service...
Aug  7 19:42:24.564898 str-s6100-acs-4 NOTICE admin: Warm boot flag: radv false.
Aug  7 19:42:24.572604 str-s6100-acs-4 NOTICE admin: Fast boot flag: radv false.
Aug  7 19:42:26.998240 str-s6100-acs-4 NOTICE admin: Started radv service...

$ sudo systemctl status radv
â— radv.service - Router advertiser container
   Loaded: loaded (/lib/systemd/system/radv.service; enabled-runtime; vendor preset: enabled)
   Active: active (running) since Fri 2020-08-07 19:43:31 UTC; 2s ago
  Process: 9817 ExecStartPre=/usr/bin/radv.sh start (code=exited, status=0/SUCCESS)
 Main PID: 9890 (radv.sh)
    Tasks: 11 (limit: 4915)
   Memory: 23.7M
   CGroup: /system.slice/radv.service
           â”œâ”€9890 /bin/bash /usr/bin/radv.sh wait
           â””â”€9891 docker wait radv
```

**- Which release branch to backport (provide reason below if seleted)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
